lib/service/db.go

```diff
@@ -30,17 +30,17 @@ import (
 	"github.com/gravitational/trace"
 )
 
-func (process *TeleportProcess) initDatabases() {
+func (process *TeleportProcess) initDatabases(initConf initConfig) {
```

Comment:
Given that we're likely to support a similar feature on other clouds, and a single service can only run on one cloud at a time, maybe a functional option pattern would be more future proof here?

`p.initDatabases(withEC2Labels())`

or 

`p.initDatabases(withGCPLabels())`

for example